mac/deploy_mac.sh

```diff
@@ -70,8 +70,22 @@ build_installer_image()
     # Install create-dmg via brew. brew needs to be installed first.
     # Download and later install. This is done to make caching possible
     local create_dmg_version="1.0.9"
-    brew extract --version="${create_dmg_version}" create-dmg homebrew/cask
-    brew install /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Formula/create-dmg@"${create_dmg_version}".rb
+    local brew_bottle_dir="${HOME}/Library/Cache/jamulus-homebrew-bottles"
+    local formula="/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Formula/create-dmg@${create_dmg_version}.rb"
+    echo "Installing create-dmg@${create_dmg_version}"
+    mkdir -p "${brew_bottle_dir}"
+    pushd "${brew_bottle_dir}"
+    if ! find . | grep -qF "create-dmg@${create_dmg_version}--"; then
+        echo "Building fresh create-dmg@${create_dmg_version} package"
+        brew developer on  # avoids a warning
+        brew extract --version="${create_dmg_version}" create-dmg homebrew/cask
+        brew install --build-bottle --formula "${formula}"
+        brew bottle "${formula}"
```

Comment:
Isn't it duplicate work if we run reinstall?

Why can't we just use an if/else?

Reply:
The problem is that `--build-bottle` seems to disable postinst scripts. I'm not sure if `aqt` uses those at all, but it might at some point, so I'd rather be on the safe side here.
There's also the option of running the postinst scripts explicitly. I chose the reinstall path though as it most closely ensures that both paths are really similar.

I should probably add a comment though.

Side note: The PR still doesn't work as it should, therefore it's still a draft again. When testing with a fake build (no compile), the logic works, while in the real build it doesn't (= always a fresh build). I'm probably missing something.

Reply:
Changed that part now to make it clearer (uninstall package & add comment + regular install later).
Testing still not finalized, will continue tomorrow.